### PR TITLE
CHE-1054: [dashboard] fix console log errors on project creation

### PR DESCRIPTION
**Pull request by akurinnoy**
**_1 Upvote**_ - github authentication request send when tab is active
- fix 'src' attribute with angular markup

### CODE REVIEW

File: dashboard/src/app/projects/create-project/github/create-project-github.controller.js [hunk 1]
```diff
@@ -58,9 +54,28 @@ export class CreateProjectGithubCtrl {
     this.state = 'IDLE';
     this.isGitHubOAuthProviderAvailable = false;
 
-    this.cheAPI.getOAuthProvider().fetchOAuthProviders().then(() => {
+    let OAuthProviderPromise = this.cheAPI.getOAuthProvider().fetchOAuthProviders().then(() => {
```
Comment (benoitf) [inline dashboard/src/app/projects/create-project/github/create-project-github.controller.js]: lowercase for the name of the variable (first letter)

Comment (akurinnoy) [inline dashboard/src/app/projects/create-project/github/create-project-github.controller.js]: fixed

Comment (benoitf) [inline dashboard/src/app/projects/create-project/github/create-project-github.controller.js]: +1

**Comment by ashumilova** [comment]
+1

**Pull request merged**